rpc/cloudagent/compute/virtualmachine/moc_cloudagent_virtualmachine.proto

```diff
@@ -96,9 +96,16 @@ message UefiSettings {
 	bool secureBootEnabled = 1;
 }
 
+enum SecurityType {
```

Comment:
Can we add this to https://github.com/microsoft/moc/blob/main/rpc/common/moc_common_computecommon.proto. This way we can avoid duplicate definitions

Reply:
Makes sense ill give it a go

Reply:
Note this triggers changes in other repros, i will update those Pr's after this completes

Reply:
Pr's are updated